src/test/java/core/basesyntax/service/impl/CsvReaderTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.Reader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CsvReaderTest {
+    private static final String VALID_PATH_TO_INPUT_FILE = "src/test/resources/testFiles/input.csv";
+    private static final String WRONG_FORMAT_FILE = "src/test/resources/testFiles/input.txt";
+    private static final String INVALID_PATH_TO_INPUT_FILE =
+            "src/test/resources/testFiles/awea" + ".csv";
+    private static Reader csvReader;
+
+    @BeforeAll
+    static void beforeAll() {
+        csvReader = new CsvReader();
+    }
+
+    @Test
+    void readFile_validPath_Ok() {
+        List<String> expected;
+        List<String> actual;
+        try {
+            expected = Files.readAllLines(Path.of(VALID_PATH_TO_INPUT_FILE));
+            actual = csvReader.readFile(VALID_PATH_TO_INPUT_FILE);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read file " + VALID_PATH_TO_INPUT_FILE);
+        }
```

Comment:
Actual must handled in reader, expected data you don`t need read from file, you can save it in String